### PR TITLE
Payments pallet blog post

### PR DESCRIPTION
**Pull request by cheqian0602**

### CODE REVIEW

File: blog/01_payments-pallet-grant.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+expectation **Virto Payment Pallet** was born. 
+
+**Virto Payment Pallet** tries to tell people a story of trust, which starts
+with **trustless**. Imagine you want to start your own business, what would be
+the most challenging part? Many answers may already come to mind, among which
+the word **trust** appears often. For example, how could you build the trust
+with your business partners? How could you gain trust from your customers every
+time they purchase from you? How could your trust other services and goods
+whenever you buy? We know it's not easy to start with just naive trust, that's
+why Virto purpose to build the Payment Pallet with the trustless design. People
```
Comment (takahser) [inline blog/01_payments-pallet-grant.md:43]:
```suggestion
why Virto's purpose is to build the Payment Pallet with a trustless design. People
```

File: blog/01_payments-pallet-grant.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+could pay each other easily using the pallet. The payment from buyers will
+always be locked first and only be released to sellers once the buyers
+confirmed to have received the product or services. The payment pallet also
+allows the creator of the payment to trigger cancel with a buffer time and the
+recipient to dispute the payment request of sender.   
+
+How does Virto Payments Pallet achieve the desired functionality? As the
+fundamental part of [LIP][lip], it's an escrow like system that allows people
+pay with cryptocurrency for any product or service from the real world, it has
```
Comment (takahser) [inline blog/01_payments-pallet-grant.md:52]:
```suggestion
to pay with cryptocurrency for any product or service from the real world, it has
```

File: blog/01_payments-pallet-grant.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+been redesigned from the ground up to be highly configurable to serve the needs
+of the Virto Network as well as any other Substrate based chain that wants to
+engage with real-world assets.
+
+We are very excited about what this milestone represents for the future of
+Virto Network, payments pallet is just the first step in our path towards
+making blockchain technology usable in the day to day live of common users and
```
Comment (takahser) [inline blog/01_payments-pallet-grant.md:59]:
```suggestion
making blockchain technology usable in the day to day life of common users and
```

File: blog/01_payments-pallet-grant.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+making _decentralized commerce with social impact_ the default way of conducting
+buissiness in the world of Web3.
```
Comment (takahser) [inline blog/01_payments-pallet-grant.md:61]:
```suggestion
business in the world of Web3. 
```

File: blog/01_payments-pallet-grant.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+time they purchase from you? How could your trust other services and goods
+whenever you buy? We know it's not easy to start with just naive trust, that's
+why Virto purpose to build the Payment Pallet with the trustless design. People
+could pay each other easily using the pallet. The payment from buyers will
+always be locked first and only be released to sellers once the buyers
+confirmed to have received the product or services. The payment pallet also
+allows the creator of the payment to trigger cancel with a buffer time and the
+recipient to dispute the payment request of sender.   
+
+How does Virto Payments Pallet achieve the desired functionality? As the
```
Comment (takahser) [inline blog/01_payments-pallet-grant.md:50]: You use different terms for the same concept:

-  Payment Pallet
- Payments Pallet
- payments pallet

It'd be more consistent to use the same term everywhere, whenever the same concept is describe.

**Review by takahser — CHANGES_REQUESTED** [review]

File: blog/01_payments-pallet-grant.md (hunk 5 above)
Comment (olanod) [reply blog/01_payments-pallet-grant.md:50]: Thanks, yeah the correct way would be "Payments Pallet", we'll look into that

**Review by olanod — APPROVED** [review]

**Pull request merged**